### ISSUE TITLE
TS-Jest version is breaking "jovo new" for Typescript

### ISSUE THREAD

**Issue by allanmelo** [jovotech/jovo-templates#61]
Error:
When trying to create a new project with the New CLI in Typescript, the dependency libs conflict. Jovo-sample-voice-app uses version 26.6.3 of Jest. However, TS-Jest supports only up to version 25.5.1.

Evidence:
<img width="603" alt="Captura de Tela 2021-02-22 às 16 15 44" src="https://user-images.githubusercontent.com/16151423/108758385-ca1ded80-7529-11eb-8e69-7eacdaf2ffce.png">

To reproduce error:
```
jovo new test --language typescript
```

Solution:
Update TS-Jest to a newer version: 26.5.1

**Comment by aswetlow** [comment]
Hey @allanmelo 

I couldn't reproduce it. 

Nevertheless, the upgrade to 26.5.1 won't break anything. I will do some tests and merge your PR. Thank you

**Comment by allanmelo** [comment]
Thanks @aswetlow. It could be something about the compatibility of libs with my dev environment. I'm using macOS Catalina 10.15.7 (19H2), node v14.15.3. Anyway, from what I tested here, nothing really broke, so it's not something that should be problematic.

It doesn't hurt to mention, other libs have also had updates. I updated everything using [npm-check](https://github.com/dylang/npm-check) and this updated version is running smoothly, with no problems here.
